packages/web-react/src/components/UNSTABLE_PartnerLogo/README.md

```diff
@@ -0,0 +1,85 @@
+# UNSTABLE PartnerLogo
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+PartnerLogo is a component designed to display the partner's logo (e.g. advertiser, business partner, etc.).
+
+```jsx
+import { UNSTABLE_PartnerLogo } from '@lmc-eu/spirit-web-react';
+
+<UNSTABLE_PartnerLogo aria-label="Logo of the partner">
+  <!-- Logo go here -->
+</UNSTABLE_PartnerLogo>
+```
+
+## Sizes
+
+The PartnerLogo component is available in [sizes][dictionary-size].
+
+```jsx
+<UNSTABLE_PartnerLogo size="small" aria-label="Logo of the partner">
+  <!-- Logo go here -->
+</UNSTABLE_PartnerLogo>
+<UNSTABLE_PartnerLogo size="medium" aria-label="Logo of the partner">
+  <!-- Logo go here -->
+</UNSTABLE_PartnerLogo>
+<UNSTABLE_PartnerLogo size="large" aria-label="Logo of the partner">
+  <!-- Logo go here -->
+</UNSTABLE_PartnerLogo>
+```
+
+## Without safe zone
+
+The PartnerLogo component can be displayed without the safe zone (padding). Use `withoutSafeZone` prop to disable safe zone around logo.
+
+```jsx
+<UNSTABLE_PartnerLogo aria-label="Logo of the partner" withoutSafeZone>
+  <!-- Logo go here -->
+</UNSTABLE_PartnerLogo>
+```
+
+## Content
+
+The content of the PartnerLogo component can be an image or svg.
+
+### Image
+
+```jsx
+<UNSTABLE_PartnerLogo aria-label="Logo of the partner">
+  <img src="path-to-logo" alt="Partner Logo" aria-hidden="true" />
+</UNSTABLE_PartnerLogo>
+```
+
+ℹ️ Don't forget to add the `aria-label` attribute for accessible title.
+The image should have an `alt` attribute set and can be aria-hidden, because the `aria-label`
+attribute is set on the container.
+
+### SVG
+
+```jsx
+<UNSTABLE_PartnerLogo aria-label="Logo of the partner">
+  <svg width="300" height="130">
+    <rect width="200" height="100" x="10" y="10" rx="20" ry="20" fill="#fff" />
+  </svg>
+</UNSTABLE_PartnerLogo>
+```
+
+ℹ️ Don't forget to add the `aria-label` attribute for accessible title.
+
+## API
+
+| Name              | Type                               | Default  | Required | Description                                             |
+| ----------------- | ---------------------------------- | -------- | -------- | ------------------------------------------------------- |
+| `children`        | `ReactNode`                        | `null`   | ✓        | Content of the PartnerLogo                              |
+| `size`            | [Size dictionary][dictionary-size] | `medium` | ✕        | Size of the PartnerLogo                                 |
+| `withoutSafeZone` | `boolean`                          | `false`  | ✕        | If true, the PartnerLogo is displayed without safe zone |
```

Comment:
We use `is` or  `has` prefix for props that are Boolean to better indicate that you can toggle some feature with it. So maybe `hasSafeZone` will be better. Image the usage `<ProductLogo hasSafeZone />`. `<ProductLogo withoutSafeZone={true} />` is not that readable. Maybe using `enable` or `disable` prefix is also better solution.

Reply:
Renamed to `disableSafeZone`

Reply:
**TLDR: `disableSafeZone` → `hasSafeArea`**

I would rather stick with `has` prefix that is frequently used in other components where property changes visual and not functionality (eg. `Tooltip` has properties `enableFlipping` or `enableSizing` which set configuration of FloatingUI).

And speaking of proper naming, I'm afraid that there is a semantic difference between `zone` and `area`.
[Safe zone](https://www.google.com/search?q=safe+zone) is mostly related to the part of the land, etc.
[Safe area](https://www.google.com/search?q=safe+area) is commonly used in digital technologies to define the area that will be always visible (eg. on screens, etc.).

Reply:
We agreed to use prop `hasSafeAreaDisabled` to prevent refactoring of CSS logic.

Also, we would like to discuss this topic with @crishpeen in the next week.